LOCALAPP.md

```diff
@@ -120,3 +120,31 @@ Currently we have not been able to provide a 100% common setup between Windows a
 The app and local platform services are now running locally. The app can be accessed on <http://altinn3local.no>.
 
 Log in with a test user, using your app name and org name. This will redirect you to the app.
+
+### Configuration of test data
+
+In some cases your application might differ from the default setup and require custom changes to the test data available. 
+This section contains the most common changes required by the app developers in localtest.
+
+#### Adjust authentication level of logged in test user
```

Comment:
```suggestion
#### Adjust authentication level of logged in test user
This would be required if your app requires a higher than default authentication level. You can also use this to give the user a lower authentication level if you want to test the app behaviour for those.
```